<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Replay</class>
 <widget class="QDialog" name="Replay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>359</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>471</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLineEdit" name="Replay_Title">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Titre de la video</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Replay_Video">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Selectionner Video</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Replay_Thumbnail">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>191</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Selectionner Thumbnail</string>
     </property>
    </widget>
    <widget class="QLabel" name="Replay_Video_Path">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>140</y>
       <width>721</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Video Path : </string>
     </property>
    </widget>
    <widget class="QLabel" name="Replay_Thumbnail_Path">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>180</y>
       <width>251</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Replay_Cancel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Replay_Submit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>250</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QLabel" name="Background_pic">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>720</width>
       <height>480</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>Background_pic</zorder>
    <zorder>Replay_Title</zorder>
    <zorder>label_2</zorder>
    <zorder>Replay_Video</zorder>
    <zorder>Replay_Thumbnail</zorder>
    <zorder>Replay_Video_Path</zorder>
    <zorder>Replay_Thumbnail_Path</zorder>
    <zorder>Replay_Cancel</zorder>
    <zorder>Replay_Submit</zorder>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
